src/main/java/DukeException.java

```diff
@@ -0,0 +1,5 @@
+public class DukeException extends Exception {
```

Comment:
Same as the question in `Duke.java`. Perhaps we can put classes in a package `exception` etc.